automatic/dropbox/tools/Helpers.psm1

```diff
@@ -0,0 +1,25 @@
+
+function Install {
+param(
+    [string] $packageName,
+    [string] $url,
+    [string] $checksum
+)
+    $installerType = 'exe'
+    $validExitCodes = @(0)
+    $silentArgs = '/S'
+    $checksumType = 'sha256';
+
+    Install-ChocolateyPackage $packageName `
+        $installerType `
+        $silentArgs `
+        $url `
+        $url `
+        -validExitCodes $validExitCodes `
+        -Checksum64 "$checksum" `
+        -ChecksumType64 $checksumType `
+        -ChecksumType "$checksumType" `
+        -Checksum "$checksum"
+}
+
+Export-ModuleMember *
```

Comment:
To you, probably none, I've been creating a bunch of packages, and having common functionality in one place means I can just re-use the helpers module across packages.